src/test/scala/BaseZioDirectMacroSupportTest.scala

```diff
@@ -69,21 +65,33 @@ class ZioDirectMacroSupportTest extends TypeInferenceTestBase {
          |  a
          |}
          |""".stripMargin,
-      "ZIO[ConfigA,Throwable,String]"
+      "ZIO[ConfigA,java.lang.Throwable,java.lang.String]"
```

Comment:
I couldn't figure out the cause, but it now infers the java types, instead of aliases in Scala's `Predef`. I've included a `testAliases` test to make sure aliases still work as expected for user code.